src/main/kotlin/oop/Composite/Cooker.kt

```diff
@@ -0,0 +1,57 @@
+package oop.Composite
+
+interface Cooker {
+  fun cook()
+}
+
+class ChineseCooker(private val cookers: MutableList<Cooker> = mutableListOf()) : Cooker {
```

Comment:
You can see an example here https://github.com/CloudCoders/Design-Patterns/pull/23/commits/016e9345fcd9bd4e8be8d53f87befb97dcf5e0e9#diff-2d06d22093fdc986cc4767c7cb391508R13